modules/ROOT/pages/getting-started-libvirt.adoc

```diff
@@ -34,6 +34,8 @@ NOTE: `virt-install` requires both the OS image and Ignition file to be specifie
 
 NOTE: Depending on your version of `virt-install`, you may not be able to use `--os-variant=fedora-coreos-*` and will get an error. In this case, you should pick an older Fedora variant (`--os-variant=fedora31` for example). You can find the variants that are supported by you current version of `virt-install` with `osinfo-query os | grep '^\s*fedora'`.
 
+NOTE: `virt-install` on Fedora 36 required additional steps:  Give proper selinux permission to ign-file: `chcon -t svirt_home_t /path/to/example.ign` and add `--network bridge=virbr0`
```

Comment:
Any specific reason for specifying explicitly the network?

Reply:
The reason for explicitly specifying the network is "default network not found" error.  Maybe just adding a link to "Booting Fedora CoreOS" would be enough, since needed parameters and steps are specified there.

Reply:
How indeed. Let's use the same ones here.